### ISSUE TITLE
Switch to disable dashboard logging

### ISSUE THREAD

**Issue by rustd** [Azure/azure-webjobs-sdk#563]
Each time the queue is processed, the SDK logs this to the dashboard. This can have a perf impact for some workloads

**Comment by JacobSnyder** [comment]
FYI: Setting `JobHostConfiguration.DashboardConnectionString` to `null` disables the dashboard logging

**Comment by mathewc** [comment]
Yes, thanks Jacob. We were just wondering whether it might be more discoverable to have a config switch to turn on/off logging.
